tablecodec/tablecodec_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/pingcap/tidb/util/codec"
 	"github.com/pingcap/tidb/util/testleak"
 	"github.com/pingcap/tidb/util/types"
+	"math"
```

Comment:
The imports are not in good order. We should separate std packages from 3rd-party packages.